extensions/ql-vscode/src/cli.ts

```diff
@@ -650,6 +650,14 @@ export class CodeQLCliServer implements Disposable {
       'Resolving queries',
     );
   }
+
+  async generateDil(qloFile: string, outFile: string): Promise<void> {
+    await this.runCodeQlCliCommand(
+      ['query', 'decompile'],
+      ['-o', outFile, qloFile],
```

Comment:
I suggest explicitly passing `--kind dil`. We do always set `includeDilInQlo` in the compilation request so this is the default, but good to be obvious.

Reply:
Sure